Userland/Utilities/CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ target_link_libraries(open LibDesktop LibMain)
 target_link_libraries(pape LibGUI LibMain)
 target_link_libraries(passwd LibCrypt LibMain)
 target_link_libraries(paste LibGUI LibCore LibMain)
+target_link_libraries(pathchk LibMain)
```

Comment:
If you're linting CMake files, please put that into another commit.

Reply:
Fixed.